### PR TITLE
BIGTOP-2869: Bump solr to 6.6.0

### PR DESCRIPTION
**Pull request by JunHe77**
Change-Id: I0b0f21e749d3638b69f9dcb9e0439cd9646ce804
Signed-off-by: Jun He <jun.he@arm.com>

### CODE REVIEW

File: bigtop-packages/src/common/solr/install_solr.sh [hunk 1]
```diff
@@ -158,136 +135,17 @@ cp -ra ${BUILD_DIR}/example/ $PREFIX/$DOC_DIR/
...
-
 # precreating /var layout
 install -d -m 0755 $VAR_DIR/log/solr
 install -d -m 0755 $VAR_DIR/run/solr
 install -d -m 0755 $VAR_DIR/lib/solr
+
+# setup solr-home
+cp -a ${BUILD_DIR}/example/resources/log4j.properties $VAR_DIR/lib/solr
+cp -a ${BUILD_DIR}/server/solr/solr.xml $VAR_DIR/lib/solr
+cp -a ${BUILD_DIR}/server/solr/zoo.cfg $VAR_DIR/lib/solr
```
Comment (evans-ye) [inline bigtop-packages/src/common/solr/install_solr.sh]: AFAIK, configs should go to /etc/solr/conf

**Comment by asanjar** [comment]
there is a conflict in the pull request..

**Comment by JunHe77** [comment]
Hi, Asanjar,

Could you pls point it? I didn't find it the conflict with master branch. Thanks.

**Comment by JunHe77** [comment]
Close this as it has been merged.

**Pull request closed without merging**